### ISSUE TITLE
deferred error : (wrong-type-argument integerp nil)

### ISSUE THREAD

**Issue by mahuron** [tkf/emacs-jedi#192]
All packages are latest from MELPA.

To reproduce failure:
```python
class Test():
    def method(self):
        pass
x = Test()
x.method(
```

The argument completion at final paren fails with _deferred error : (wrong-type-argument integerp nil)_ .  Disabling jedi-mode prevents the error.

Server debug:
```
receiving...
received: length = 114; data = '(call 249 get_in_function_call ("class Test():\n    def method(self):\n        pass\nx = Test()\nx.method(" 1 0 nil))\n'
(AutoLog) Called: EPCHandler._handle('(call 249 get_in_function_call ("class Test():\n    def method(self):\n        pass\nx = Test()\nx.method(" 1 0 nil))\n')
(AutoLog) Called: EPCHandler._handle_call(249, Symbol(u'get_in_function_call'), [u'class Test():\n    def method(self):\n        pass\nx = Test()\nx.method(', 1, 0, []])
(AutoLog) Returns: EPCHandler._handle_call(...) = ['return', 249, []]
(AutoLog) Called: EPCHandler._send('return', 249, [])
(AutoLog) Returns: EPCHandler._send(...) = None
(AutoLog) Returns: EPCHandler._handle(...) = None
```

**Comment by syohex** [comment]
Which Emacs version do you use ?

**Comment by syohex** [comment]
@mahuron  Thanks for information. I fixed this issue at #193.
Please check latest version.

**Comment by mahuron** [comment]
Latest pull from MELPA has resolved this.  Thank you.
